ruby_event_store/spec/in_memory_repository_spec.rb

```diff
@@ -8,9 +8,32 @@ module RubyEventStore
     let(:test_race_conditions_any)   { false }
     let(:test_race_conditions_auto)  { true }
     let(:test_expected_version_auto) { true }
+    let(:test_link_events_to_stream) { true }
```

Comment:
I don't see where it used.

Reply:
> Large diffs are not rendered by default.

Press "load diff" and check hundreds of new lines of tests in `ruby_event_store/lib/ruby_event_store/spec/event_repository_lint.rb` :)